### ISSUE TITLE
[69] Prints to stdout if cannot load dot_parser

### ISSUE THREAD

**Issue by prmtl** [pydot/pydot-ng#10]
**Reported by cmr.P...@gmail.com, 2012-03-20T11:05:59Z**

If dot_parser cannot be loaded in pydot, a message is printed to stdout. I believe it's incorrect. A message should be printed to stderr (as in the following patch), or not printed at all.

```
*** pydot.py.orig   2012-03-20 13:48:01.009730200 +0400
--- pydot.py    2012-03-20 13:48:27.784195800 +0400
***************
*** 23,28 ****
--- 23,29 ----
  __license__ = 'MIT'

  import os
+ import sys
  import re
  import subprocess
  import tempfile
***************
*** 30,36 ****
  try:
      import dot_parser
  except Exception, e:
!     print "Couldn't import dot_parser, loading of dot files will not be possible."



--- 31,37 ----
  try:
      import dot_parser
  except Exception, e:
!     print >> sys.stderr, "Couldn't import dot_parser, loading of dot files will not be possible."
```

_Originally assigned to ero.carr...@gmail.com_
From: https://code.google.com/p/pydot/issues/detail?id=69

**Comment by prmtl** [comment]
**Comment by michal.b...@gmail.com, 2013-08-02T06:51:11Z**

smells like a bug, is a bug (i don't want to have unneeded module import failure message in my resulting data output)

**Comment by prmtl** [comment]
**Comment by ero.carr...@gmail.com, 2014-03-15T16:06:26Z**

Yes, the error should be printed to stderr. It will be fixed.

**Comment by prmtl** [comment]
**Comment by ero.carr...@gmail.com, 2014-03-15T16:06:43Z**

*Assigned to ero.carr...@gmail.com*

**Comment by prmtl** [comment]
Fixed in #48 by using warnings module.
